interpreter/cling/lib/Utils/Validation.cpp

```diff
@@ -18,12 +19,41 @@
 # include <unistd.h>
 #endif
 
-
+#define CACHESIZE 8
```

Comment:
Please consider 'constexpr auto kCacheSize = 8;' and maybe define it in the class that needs it.